docs/admin/install/knative-with-operators.md

```diff
@@ -199,12 +199,58 @@ knative-serving   <version number>    True
 
 ### Installing with different networking layers
 
-??? "Installing the Knative Serving component with different network layers"
+!!! note "Installing the Knative Serving component with different network layers"
 
     Knative Operator can configure Knative Serving component with different network layer options. Istio is the default network
     layer, if the ingress is not specified in the Knative Serving CR. Click on each of the following tabs to see how you can configure
     Knative Serving with different ingresses:
 
+    === "Istio (default)"
+
+        The following commands install Istio and enable its Knative integration.
+
+        1. [Install Istio](installing-istio.md).
+
+        1. To configure Knative Serving to use Istio, copy the following YAML into a file:
+          ```yaml
+          apiVersion: operator.knative.dev/v1alpha1
+          kind: KnativeServing
+          metadata:
+            name: knative-serving
+            namespace: knative-serving
+          ```
+
+        1. If Istio is installed under a namespace other than the default `istio-system`, you need to change the content
+           of the YAML into:
+          ```yaml
+          apiVersion: operator.knative.dev/v1alpha1
+          kind: KnativeServing
+          metadata:
+            name: knative-serving
+            namespace: knative-serving
+          spec:
+            config:
+              istio:
+                local-gateway.<local-gateway-namespace>.knative-local-gateway: "knative-local-gateway.<istio-namespace>.svc.cluster.local"
+          ```
+
+          Where `<local-gateway-namespace>` is the local gateway namespace, which is the same as Knative Serving
+          namespace `knative-serving`, and <istio-namespace> is the namespace, where Istio is installed.
```

Comment:
```suggestion
              Where:
          
              - `<local-gateway-namespace>` is the local gateway namespace, which is the same as Knative Serving namespace `knative-serving`.
              - `<istio-namespace>` is the namespace where Istio is installed.
```

Reply:
I'm not clear about what line 239 means:
> ... the local gateway namespace, which is the same as Knative Serving namespace `knative-serving`


Does this mean that `<local-gateway-namespace>` will always be `knative-serving`?

Reply:
<local-gateway-namespace> should be the same as the namespace for the knative serving. for knative serving installed with the operator, it is defaulted to knative-serving.